libs/base/configkeys.h

```diff
@@ -212,6 +212,31 @@
 #define CFG_ANALOG_JOYSTICK_MIN 216
 #define CFG_ANALOG_JOYSTICK_MAX 217
 #define CFG_TIMERS_TO_USE 218
+#define CFG_PIN_ONBOARD_DOTSTAR_CLOCK 219
+#define CFG_PIN_ONBOARD_DOTSTAR_DATA 220
```

Comment:
why do we have separate neopixel pin and onboard neopixel?

Reply:
Added notes. Note this change is backward compat with pybadge.